From 406d0de6f87f3d300ab09fb79ed821d772d829ab Mon Sep 17 00:00:00 2001
From: Adrien Mazarguil <adrien.mazarguil@6wind.com>
Date: Thu, 1 Feb 2018 14:30:57 +0100
Subject: [PATCH 6/9] net/mlx: fix missing includes for rdma-core glue

For consistency since these includes are already pulled by others.

Fixes: 6aca97d310 ("net/mlx4: move rdma-core calls to separate file")
Fixes: 7202118686 ("net/mlx5: move rdma-core calls to separate file")

Signed-off-by: Adrien Mazarguil <adrien.mazarguil@6wind.com>
(cherry picked from commit 0138edd60cb13612bc7ed638a4d549bcc60008b2)
---
 drivers/net/mlx4/mlx4_glue.c | 3 +++
 drivers/net/mlx4/mlx4_glue.h | 3 +++
 drivers/net/mlx5/mlx5_glue.h | 1 +
 3 files changed, 7 insertions(+)

diff --git a/drivers/net/mlx4/mlx4_glue.c b/drivers/net/mlx4/mlx4_glue.c
index 30797bd..47ae7ad 100644
--- a/drivers/net/mlx4/mlx4_glue.c
+++ b/drivers/net/mlx4/mlx4_glue.c
@@ -3,6 +3,9 @@
  * Copyright 2018 Mellanox
  */
 
+#include <stddef.h>
+#include <stdint.h>
+
 /* Verbs headers do not support -pedantic. */
 #ifdef PEDANTIC
 #pragma GCC diagnostic ignored "-Wpedantic"
diff --git a/drivers/net/mlx4/mlx4_glue.h b/drivers/net/mlx4/mlx4_glue.h
index 0623511..de251c6 100644
--- a/drivers/net/mlx4/mlx4_glue.h
+++ b/drivers/net/mlx4/mlx4_glue.h
@@ -6,6 +6,9 @@
 #ifndef MLX4_GLUE_H_
 #define MLX4_GLUE_H_
 
+#include <stddef.h>
+#include <stdint.h>
+
 /* Verbs headers do not support -pedantic. */
 #ifdef PEDANTIC
 #pragma GCC diagnostic ignored "-Wpedantic"
diff --git a/drivers/net/mlx5/mlx5_glue.h b/drivers/net/mlx5/mlx5_glue.h
index 67bd8d0..a2cd18e 100644
--- a/drivers/net/mlx5/mlx5_glue.h
+++ b/drivers/net/mlx5/mlx5_glue.h
@@ -6,6 +6,7 @@
 #ifndef MLX5_GLUE_H_
 #define MLX5_GLUE_H_
 
+#include <stddef.h>
 #include <stdint.h>
 
 /* Verbs headers do not support -pedantic. */
-- 
1.8.3.1

